dashboard/modules/log/log_agent.py

```diff
@@ -296,7 +296,7 @@ async def ListLogs(self, request, context):
             )
         log_files = []
         for p in path.glob(request.glob_filter):
-            log_files.append(str(p.relative_to(path)))
+            log_files.append(str(p.relative_to(path)) + ("/" if p.is_dir() else ""))
```

Comment:
does it work with nested directory inside log dir btw?

Reply:
(it doesn't have to work imo, but i'd like to just know)

Reply:
yup, I tested this